src/webhooks/pubsub/slackNotifications.ts

```diff
@@ -248,162 +247,157 @@ export const constructSlackMessage = (
+      hasEnoughTimePassedSinceIssueCreation;
+    const formattedIssues = sortAndFlattenIssuesArray(triageQueueIssues);
+    if (
+      triageQueueIssues.length > 0 &&
+      overdueIssues.length === 0 &&
+      actFastIssues.length === 0
+    ) {
+      messageBlocks.push({
+        type: 'section',
+        fields: [
+          {
+            type: 'mrkdwn',
+            text: `⏳ *Triage Queue*`,
+          },
+          { type: 'mrkdwn', text: `😯` },
+        ],
+      });
+      for (let i = 0; i < formattedIssues.length; i += 2) {
         messageBlocks.push({
           type: 'section',
-          fields: [
-            {
-              type: 'mrkdwn',
-              text: `⏳ *Triage Queue*`,
-            },
-            { type: 'mrkdwn', text: `😯` },
-          ],
+          fields: [formattedIssues[i], formattedIssues[i + 1]],
         });
-        for (let i = 0; i < formattedIssues.length; i += 2) {
-          messageBlocks.push({
-            type: 'section',
-            fields: [formattedIssues[i], formattedIssues[i + 1]],
-          });
-        }
-      }
-      /*
-        Two cases to skip sending message
-        1. No issues in any queue
-        2. Issues are in triage queue but channel doesn't need to be notified
-      */
-      if (
-        (overdueIssues.length === 0 &&
-          actFastIssues.length === 0 &&
-          triageQueueIssues.length === 0) ||
-        (overdueIssues.length === 0 &&
-          actFastIssues.length === 0 &&
-          triageQueueIssues.length > 0 &&
-          !shouldNotifyForOnlyTriagedQueue)
-      ) {
-        return Promise.resolve();
       }
-      return bolt.client.chat
-        .postMessage({
-          channel: channelId,
-          text: '👋 Triage Reminder ⏰',
-          blocks: messageBlocks,
-        })
-        .then(async () => {
-          await getChannelLastNotifiedTable()
-            .insert({ channel_id: channelId, last_notified_at: now })
-            .onConflict('channel_id')
-            .merge();
-        });
     }
-    return Promise.resolve();
+    /*
+      Two cases to skip sending message
+      1. No issues in any queue
+      2. Issues are in triage queue but channel doesn't need to be notified
+    */
+    if (
+      (overdueIssues.length === 0 &&
+        actFastIssues.length === 0 &&
+        triageQueueIssues.length === 0) ||
+      (overdueIssues.length === 0 &&
+        actFastIssues.length === 0 &&
+        triageQueueIssues.length > 0 &&
+        !shouldNotifyForOnlyTriagedQueue)
+    ) {
+      return Promise.resolve();
+    }
+    return bolt.client.chat
+      .postMessage({
+        channel: channelId,
+        text: '👋 Triage Reminder ⏰',
+        blocks: messageBlocks,
+      })
+      .then(async () => {
+        // Would be helpful to have some logs around sending slack triage notifications for debugging
+        // eslint-disable-next-line no-console
+        console.log(`Sent slack triage message to ${channelId}.`);
```

Comment:
Yes, I know this is not ideal, but I tested this locally using the exact same payload as I should get in prod and it works. Not sure why it's not sending in prod and this will help to determine why

Reply:
nit: I would recommend leaving a TODO pointed at a bug so we don't miss this.